libs/components/forms/src/lib/modules/selection-box/selection-box-grid.component.scss

```diff
@@ -32,15 +84,57 @@
     }
```

Comment:
was there any rhyme or reason to the max width/flex numbers above are or they just magic numbers? Is the flex one the max width minus padding or something?

Reply:
Not that I'm aware of.